### PR TITLE
Spike/jest test

### PR DESCRIPTION
**Pull request by clewis1**
https://companieshouse.atlassian.net/browse/IDVA3-1124

- Added required configuration files and removed mocha references.
- Added simple unit tests to start page and confirm company page.

More jest documentation can be found here 
https://jestjs.io/docs/getting-started

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by hepsimo — COMMENTED** [review]
Is there a convention (at CH?) to use Jest's synonym 'it' instead of 'test'? The latter seems more intuitive/expressive to me.

File: jest.config.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+module.exports = {
+    roots: ["<rootDir>"],
+    testPathIgnorePatterns: ["/node_modules/", "/dist/"],
+    collectCoverageFrom: ["./src/**/*.ts"],
```
Comment (hepsimo) [inline jest.config.js:4]: We should probably enable code coverage here

```suggestion
    collectCoverage: true,
```

File: test/routers/healthcheck.unit.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import app from "../../src/app";
+import request from "supertest";
+import { PrefixedUrls } from "../../src/constants";
+
+describe("healthcheck tests", () => {
+    it("should show status 200", async () => {
+        const resp = await request(app).get(PrefixedUrls.HEALTHCHECK);
```
Comment (hepsimo) [inline test/routers/healthcheck.unit.ts:7]: We could indicate test is not ready by skipping it

```suggestion
    it.skip("should show status 200", async () => {
```

**Review by hepsimo — APPROVED** [review]

**Review by hepsimo — APPROVED** [review]
Changes look good. Added some thoughts/suggestions

**Pull request merged**